route/build/src/globalData.f90

```diff
@@ -249,6 +255,8 @@ MODULE globalData
   real(dp),           allocatable, public :: basinEvapo_main(:)     ! HRU evaporation array (m/s) for mainstem
   real(dp),           allocatable, public :: basinPrecip_trib(:)    ! HRU precipitation array (m/s) for tributaries
   real(dp),           allocatable, public :: basinPrecip_main(:)    ! HRU precipitation array (m/s) for mainstem
+  real(dp),           allocatable, public :: basinCC_trib(:)        ! HRU constituent array (g/s) for tributaries
```

Comment:
Again as further above, point out the connection to solute.

Reply:
`basinCC_trib` is changed to `basinSolute_trib`